src/Tinkoff.ISA.API/Startup.cs

```diff
@@ -40,10 +45,31 @@ public void ConfigureServices(IServiceCollection services)
             services.AddAppDependencies();
             services.AddMvc();
             services.AddAutoMapper();
+            services.AddHangfire((serviceProvider, config) =>
```

Comment:
Seems better to move it to Tinkoff.Isa.Infrastructure project, do you agree? to avoid duplications in API, Scheduler, SchedulerUI

Reply:
Maybe we can remove it?